<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonMenu</class>
 <widget class="QWidget" name="ButtonMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="pbStatistics">
     <property name="toolTip">
      <string>Show statistics window</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/statistics2.png</normaloff>:/icons/statistics2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>47</width>
       <height>47</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbHistory">
     <property name="toolTip">
      <string>Show history window</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/streamline_history.png</normaloff>:/icons/streamline_history.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>47</width>
       <height>47</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbProgress">
     <property name="toolTip">
      <string>Show goals window</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/progress.svg</normaloff>:/icons/progress.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>47</width>
       <height>47</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbSettings">
     <property name="toolTip">
      <string>Show settings window</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/streamline_settings.png</normaloff>:/icons/streamline_settings.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>47</width>
       <height>47</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
